pom.xml

```diff
@@ -23,6 +23,7 @@
         <dependency>
           <groupId>omero</groupId>
           <artifactId>blitz</artifactId>
+          <version>5.4.10-${ice.version}-b105</omero.version>
```

Comment:
Tags don't match.

Reply:
Thanks. Fixed in d1da4af . I started modifying the `pom.xml` before realizing that Travis is testing using Gradle.